slackSupport.js

```diff
@@ -6,15 +6,19 @@ exports.slackLogin = function () {
 	console.log(chalk.magenta(`Slack Enabled... Starting.`));
 	if (Kirbi.Auth.slack.bot_token) {
 		console.log('Logging in to Slack...');
-        Kirbi.Slack = new RtmClient(Kirbi.Auth.slack.bot_token);
+		let MemoryDataStore = require('@slack/client').MemoryDataStore;
+        Kirbi.Slack = new RtmClient(Kirbi.Auth.slack.bot_token, {
+			logLevel: 'error',
+			dataStore: new MemoryDataStore()
+		});
 		Kirbi.Slack.c_events = require('@slack/client').CLIENT_EVENTS;
 		Kirbi.Slack.rtm_events = require('@slack/client').RTM_EVENTS;
         Kirbi.Slack.start();
 		require('./lib/onEvent');
 	} else {
         console.log(chalk.red('ERROR: Kirbi must have a Slack bot token...'));
         return;
-	};
+	}
```

Comment:
your doing a value assignment, I would keep the ; 

I don't know if we talked about this but I prefer ; after every statement for clarity.